packages/vue-query/src/__mocks__/useBaseQuery.ts

```diff
@@ -1,5 +1,7 @@
+import { vi } from 'vitest'
+
 const { useBaseQuery: originImpl, unrefQueryArgs: originalParse } =
-  jest.requireActual('../useBaseQuery')
+  (await vi.importActual('../useBaseQuery')) as any
 
-export const useBaseQuery = jest.fn(originImpl)
```

Comment:
If `vi.mock` automatically wraps all `original implementations` with spy functions, then we could remove it.
Jest used the `__mock__` directory for mocking modules, without necessity to write the same mock in multiple files.

Reply:
makes sense - thanks!